frontend/src/components/common/Logo/Logo.tsx

```diff
@@ -5,15 +5,17 @@ import { ReactComponent as SvgLogo } from '../../../assets/logo.svg';
 export interface ILogoProps {
   widthPx?: number;
   className?: string;
+  color?: string;
 }
 
 const Logo: FC<ILogoProps> = ({ ...props }) => {
-  const { widthPx, className } = props;
+  const { widthPx, className, color } = props;
   return (
     <div className={'flex items-center justify-center m-0 p-0 ' + className}>
```

Comment:
Are we sure we want to keep the logo with this div wrapper? (with such flex stuff too?)

Reply:
I removed it. It seems work correctly without it.